buildenv/jenkins/variables/defaults.yml

```diff
@@ -61,3 +61,20 @@ linux_390-64_cmprssptrs:
   node_labels:
     build: '390'
     test: '390'
+#========================================#
+# AIX PPC 64bits Compressed Pointers
+#========================================#
+aix_ppc-64_cmprssptrs:
+  boot_jdk:
+    8: '/usr/java7'
+    9: '/usr/java8_64'
+  release:
+    8: 'aix-ppc64-normal-server-release'
+    9: 'aix-ppc64-normal-server-release'
+  freemarker: '${HOME}/freemarker.jar'
+  node_labels:
+    build: 'aix'
+    test: 'aix'
+  extra_configure_options:
+    8: '--with-cups-include=/opt/freeware/include --with-extra-ldflags=-lpthread --with-extra-cflags=-lpthread --with-extra-cxxflags=-lpthread DF=/usr/sysv/bin/df --disable-ccache --with-jobs=8'
+    9: '--with-cups-include=/opt/freeware/include --with-extra-ldflags=-lpthread --with-extra-cflags=-lpthread --with-extra-cxxflags=-lpthread DF=/usr/sysv/bin/df --disable-warnings-as-errors --with-jobs=8'
```

Comment:
Is `--with-jobs=8` for Java 9 correct?

Reply:
ok, # of parallel jobs

Reply:
That the number of cpus available; openjdk doesn't recognize smt enablement; so hard coding is required until this is resolved at the configure level. https://github.com/ibmruntimes/openj9-openjdk-jdk8/issues/57